doc/source/whatsnew/v0.20.0.txt

```diff
@@ -388,8 +393,7 @@ For example, after running the following, ``styled.xlsx`` renders as below:
    df
    styled = df.style.\
        applymap(lambda val: 'color: %s' % 'red' if val < 0 else 'black').\
```

Comment:
I think this is the best we have right now. A `.where` would be nice... I'll make a new issue :)